### DISCUSSION TITLE
SDK implementation for C#

### DISCUSSION THREAD

**Discussion by anvimaa** [pocketbase/pocketbase#4435] [Polls]
Firstly, thank everyone in the community for the incredible work they have done with this wonderful project.

However, I would like to see an implementation of an **_SDK for C#_** projects.

**Comment by ramadhanafif** [comment]
Isn't C# users typically uses microsoft sql and the orm provided?

**Comment by softexpert** [comment]
I vote for an implementation once the refactoring is done; SDK's should be available in/for more than just Go and JavaScript - a rich SDK ecosystem ensures wider adoption and gives more integration opportunities; the drawback is that changes are more disruptive and it's harder to maintain a diverse SDK collection, but that's the beauty of communities ...

**Comment by ganigeorgiev** [comment]
This discussion is outdated and there are no plans for official C# SDK until stable v1 is reached.
Community contributions are welcomed. See also https://github.com/pocketbase/pocketbase/discussions/391.
